pom.xml

```diff
@@ -24,6 +24,7 @@
     <module>gapic-generator-java-bom</module>
     <module>java-shared-dependencies</module>
     <module>sdk-platform-java-config</module>
+    <module>java-sdk-logging</module>
```

Comment:
Is this included in releases?

Reply:
No, the module is skipped during normal release.

Reply:
Great. Do you mind point me to where the normal release is setup?

Reply:
`maven-deploy-plugin` in these `pom.xml`s are skipped.

This is the same setup in the root `pom.xml`.

Reply:
Does it have to be added to the root pom?

Reply:
No, it doesn't need to add to root pom.

Without this line, we need to specify the pom to build the module,
```
mvn clean install -f java-sdk-logging/pom.xml
```

I'll remove this line then.

Reply:
I would prefer the tests for this module are a separate CI, since it has no dependencies on other modules at all. Even if the tests fail, it should not block the release of other modules.